.github/workflows/main.yml

```
@@ -2,37 +1,68 @@
+        run: docker build -t ${{ secrets.DOCKER_USERNAME }}/material:${{ github.ref_name }} .
 
+      # Push Docker image
       - name: Push Docker image
+        run: docker push ${{ secrets.DOCKER_USERNAME }}/material:${{ github.ref_name }}
+
+      # Authenticate to GCP
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v2
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
```

actionlint (1.7.7)

input "service_account_key" is not defined in action "google-github-actions/setup-gcloud@v2". available inputs are "install_components", "project_id", "skip_install", "version"

(action)